apps/api/src/user/model-schemas/user-api-key/user-api-key.schema.ts

```diff
@@ -0,0 +1,28 @@
+import { sql } from "drizzle-orm";
+import { boolean, index, pgTable, timestamp, unique, uuid, varchar } from "drizzle-orm/pg-core";
+
+import { Users } from "@src/user/model-schemas";
+
+export const UserApiKeys = pgTable(
+  "user_api_keys",
```

Comment:
why is it `user_api_keys`? Shouldn't it just be
```suggestion
  "api_keys",
```